target/linux/ath79/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -13,6 +13,9 @@ PHYNBR=${DEVPATH##*/phy}
 board=$(board_name)
 
 case "$board" in
+	iodata,wn-ac1600dgr2)
```

Comment:
please patch the mac address in the ath10k firmware instead.

Reply:
Is it also for SoC internal wmac?

Reply:
Wait, I got confused. For which wireless you need to set the mac address? Isn't it the pcie ath10k wireless? Or does the SoC wifi has a wrong mac address as well and you don't know yet how to fix it?

Would you please paste the mac addresses of the stock firmware and provide a dump of the u-boot-env and art partition. Might be better if I have a look on it on my own before asking more questions and giving bad advices. Feel free to send it to me via mail.

Reply:
Oh, I'm sorry I confused you...
WN-AC1600DGR2 supported 2.4G (internal) and 5G (pcie) wifi, but mac addresses (ethaddr, wanaddr) is stored only in u-boot-env as far as I have investigated. So it is necessary to read from u-boot-env and set it for 2.4/5GHz wifi.

- MAC addresses
  - LAN: 34:76:C5:35:11:30
  - WAN: 34:76:C5:35:11:32
  - 2.4G: 34:76:C5:35:11:30
  - 5G: 34:76:C5:35:11:31

I sent an email with dump files attached.

Reply:
> Oh, I'm sorry I confused you...

No I failed to see that 10_fix_wifi_mac fixes the mac address of all wireless phys and that your board has a wmac as well. More will follow via e-mail.